tools/packaging/README.md

```diff
@@ -0,0 +1,40 @@
+# Installation Packages
+
+## Overview
+This directory includes a set of files and scripts to build installation package for various Linux distributions.
+
+## Debian
+The file to build the Debian package all located under "debian" directory.
+The resulting package will install the binary of Dragonfly as well as generate a new service entry for dragonfly,
+that can be controlled with "systemctl" command, to start, stop and check status of.
+### Building
+To build the package, you have a script called "generate_debian_package.sh". This script accepts the following parameters:
+* Optional binary path - the location from which to take the binary for the installation. The default for this is "repo path/build-opt".
+The location to which the resulting package is writing is at the location from which the script is executed.
+This script is depends on the following packages:
+* git
+* moreutils
+* debhelper
+* dpkg-dev
+
+To build:
+```
+/path/to/dragonfly/tools/packaging/generate_debian_package.sh [/path/to/dragonfly-binary-file]
+```
+
+This can only be run on Debian based hosts.
+You can use the flowing docker file to generate this package:
+```
+FROM ubuntu:20.04
+ENV TZ=Asia/Dubai
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
because if you have a docker without the build tools (such as gcc/make and friends), when you're building it, it will promote you to select the timezone - [see this issue](https://grigorkh.medium.com/fix-tzdata-hangs-docker-image-build-cdb52cc3360d)

Reply:
usually you just need to define `ARG DEBIAN_FRONTEND=noninteractive`